### PR TITLE
Update a100_throughputs.md

### PR DESCRIPTION
**Pull request by nixli**
* [ yes ] Tested
* [ N/A ] Formatted with YAPF

### CODE REVIEW

File: docs/mlsys21/a100_throughputs.md [hunk 1]
```diff
@@ -105,28 +105,35 @@ git clone https://github.com/UofT-EcoSystem/hfta.git
...
 ```
 
-### Acquiring NVIDIA Nsight Compute CLI and DCGM
+### Acquiring NVIDIA Nsight Systems CLI and DCGM
 
-We require two installation files (`.deb`) for Nsight Compute and DCGM pre-downloaded to build the docker image.
+We require two installation files (`.deb`) for Nsight Systems and DCGM pre-downloaded to build the docker image.
 
-- Nsight Compute: version `3.1.72`, downloaded under `third_party/nsys/nsys_cli_2020.3.1.72.deb`
+- Nsight System: version `3.1.72`, downloaded under `third_party/nsys/nsys_cli_2020.3.1.72.deb`
```
Comment (wangshangsam) [inline docs/mlsys21/a100_throughputs.md:112]: `System` -> `Systems`

File: docs/mlsys21/a100_throughputs.md [hunk 2]
```diff
@@ -105,28 +105,35 @@ git clone https://github.com/UofT-EcoSystem/hfta.git
...
 
 
 ### Download and launch docker image
 
 Follow the commands below to prepare and launch the docker image, this will take approximately 10 mins
 
-#### Build docker image
+#### Build or pull docker image
+
+You can either build the image from scratch or pull the pre-built image from docker-hub
```
Comment (wangshangsam) [inline docs/mlsys21/a100_throughputs.md:126]: `Docker Hub.`

File: docs/mlsys21/a100_throughputs.md [hunk 3]
```diff
@@ -105,28 +105,35 @@ git clone https://github.com/UofT-EcoSystem/hfta.git
...
+To build the image from scracth:
 ```bash
-# build the image, select from native1.6-cu10.2, nvidia20.06, nvidia20.08
+# build the image, select from nvidia20.06 or nvidia20.08 to ensure it works with A100
```
Comment (wangshangsam) [inline docs/mlsys21/a100_throughputs.md:129]: Let's just limit it to `nvidia20.06` for the sake of simplicity, then type out the command explicitly below:
```bash
bash docker/build.sh nvidia20.06
```

File: docs/mlsys21/a100_throughputs.md [hunk 4]
```diff
@@ -105,28 +105,35 @@ git clone https://github.com/UofT-EcoSystem/hfta.git
...
 # this will take about 10 mins to complete
 bash docker/build.sh <the version of the image, e.g. nvidia20.08>
 ```
+To pull from docker-hub
+```bash
+docker pull wangshangsam/hfta:mlsys21_<the version of the image, e.g. nvidia20.08>
```
Comment (wangshangsam) [inline docs/mlsys21/a100_throughputs.md:135]: Type out the command explicitly: 
```bash
docker pull wangshangsam/hfta:mlsys21_nvidia20.06
```

**Review by wangshangsam — CHANGES_REQUESTED** [review]

File: docs/mlsys21/a100_throughputs.md [hunk 5]
```diff
@@ -105,27 +105,35 @@ git clone https://github.com/UofT-EcoSystem/hfta.git
...
 ```bash
-# build the image, select from native1.6-cu10.2, nvidia20.06, nvidia20.08
+# build the image, select nvidia20.06 to work with the A100 GPU.
 # this will take about 10 mins to complete
-bash docker/build.sh <the version of the image, e.g. nvidia20.08>
+bash docker/build.sh nvidia20.06
+```
+To pull from Docker Hub directly without building it locally:
+```bash
+docker pull wangshangsam/hfta:mlsys21_nvidia20.06
```
Comment (wangshangsam) [inline docs/mlsys21/a100_throughputs.md:136]: Add
`docker tag wangshangsam/hfta:mlsys21_nvidia20.06 hfta:dev`

**Review by wangshangsam — APPROVED** [review]
LGTM! Only a minor change is needed.

File: docs/mlsys21/a100_throughputs.md (hunk 5 above)
Comment (nixli) [reply docs/mlsys21/a100_throughputs.md:136]: done! thank you!

**Pull request merged**